modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -349,4 +349,89 @@ The output file is:
 k/J1EHyxJagOcyQELoaqIDCzzKc1bLGJiHY2dv6jzuhtVpP1LTlgK3y8EH7OkK5iuuu8kik3P5dCDMixZXwn/EV6xbUwLFKnEb4x1pUX6aopLP/KULm0pQ==
 ----
 
-Note that key sizes might depend on the Java Cryptography Extension (JCE) version. For example, since version 1.8.0_161, the AES key size needs to be of at least 32 characters (256 bytes), instead of 16 for previous versions.
+=== Arguments Reference
```

Comment:
The first five rows in this table seem confusing. In these rows, what is the relationship between the first and the third columns?

For example, if the argument is `string`, how is it possible that `file` and `file-level`, which are separate arguments, are documented here as Accepted Values for `string`?

The three values are acceptable for decryption mode, but not for the argument `string`. But I don't know how to reword this. I think the accepted values for string, file, and file-level are really the second sentence of the description column entry:

Argument             Description                 Value
string                   Sets...                          An encrypted single-string value provided as argument <value>.

Does that make any sense?

Reply:
I'm reworking the whole to get rid of the third column and clarify difference between arguments and values.